### PR TITLE
Hero health bar

### PR DESCRIPTION
**Pull request by hotblack86**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -8,7 +8,26 @@
 from constants import (WIDTH, HEIGHT, TILESIZE, TITLE, BG_COLOUR, DARK_LINE,
                        FONT_NAME, GOLD, GAME_SPEED)
 from hitbox import Hitbox
-
+from collision import game_over_voice
+from constants import WHITE, GREEN, RED, YELLOW, BAR_LENGTH, BAR_HEIGHT
+
```
Comment (Sindex42) [inline src/main.py:13]: Could merge these constants with the rest or extract into a HUD class later?

**Review by Sindex42 — APPROVED** [review]
Nice work with Hero health 👍

**Review by CKKH — APPROVED** [review]
Awesome work James

File: src/enemy.py [hunk 2]
```diff
@@ -39,24 +39,27 @@ def move(self):
...
     def load_direction_image(self, direction):
         ''' Load directional facing sprites '''
 
         self.image = pg.transform.scale(
             pg.image.load(f'./images/{self.type}/{self.type}_{direction}.png'),
             (TILESIZE, TILESIZE)).convert()
 
-    def end_game(self):
-        ''' End game process '''
+    def enemy_touches_hero(self):
```
Comment (michaelnguyen974) [inline src/enemy.py:56]: Quality stuff

File: src/main.py [hunk 3]
```diff
@@ -5,10 +5,29 @@
...
-                       FONT_NAME, GOLD, GAME_SPEED)
 from hitbox import Hitbox
-
+from collision import game_over_voice
+from constants import (WIDTH, HEIGHT, TILESIZE, TITLE, BG_COLOUR, DARK_LINE,
+                       FONT_NAME, GOLD, GAME_SPEED, WHITE, GREEN, RED, YELLOW,
+                       BAR_LENGTH, BAR_HEIGHT)
+
+# HUD functions
+def draw_hero_health(surf, x_pos, y_pos, pct):
```
Comment (michaelnguyen974) [inline src/main.py:15]: This is good, maybe we can refactor the if else statements?

**Review by michaelnguyen974 — APPROVED** [review]

**Pull request merged**